iothub/device/src/DirectMethod/DirectMethodRequest.cs

```diff
@@ -28,86 +29,32 @@ internal DirectMethodRequest()
         /// This class can be inherited from and set by unit tests for mocking purposes.
         /// </remarks>
         /// <param name="methodName">The method name to invoke.</param>
-        protected internal DirectMethodRequest(string methodName)
+        /// <param name="payload">The direct method payload.</param>
+        protected internal DirectMethodRequest(string methodName, byte[] payload = default)
         {
             MethodName = methodName;
+            _payload = payload;
         }
 
         /// <summary>
         /// The method name to invoke.
         /// </summary>
-        [JsonProperty("methodName")]
         public string MethodName { get; private set; }
 
-        /// <summary>
-        /// The amount of time given to the service to connect to the device.
-        /// </summary>
-        /// <remarks>
-        /// A timeout may occur if this value is set to zero and the target device is not connected to
-        /// the cloud.
-        /// If the value is greater than zero, it may also occur if the cloud fails to deliver the request to
-        /// the target device.
-        /// <para>
-        /// This value is propagated to the service in terms of seconds, so this value does not have a level of
-        /// precision below seconds. For example, a value of <c>TimeSpan.FromMilliseconds(500)</c> will be
-        /// interpreted as 0 seconds (using <c>ConnectTimeout.TotalSeconds</c>).
-        /// </para>
-        /// </remarks>
-        [JsonIgnore]
-        public TimeSpan? ConnectionTimeout { get; set; }
```

Comment:
These timeouts are never set by the SDK for a device method request received from service